package.json

```diff
@@ -27,6 +26,10 @@
     "webpack": "^4.44.1",
     "webpack-cli": "^3.3.12"
   },
+  "optionalDependencies": {
+    "prop-types": "^15.7.2",
+    "react": "^17.0.0"
```

Comment:
Currently, these dependencies are inlined into the bundle, let's use [externals](https://webpack.js.org/configuration/externals/) to keep them extracted.

Reply:
I hope that will help with that, I will add it
```js
  externals: {
    'prop-types': 'prop-types',
    'react': 'react'
  },
```

Reply:
The general idea is not to hope but to test it using a bundle analyzer and trying to use it in different environments

Reply:
I will check it at Monday's monring